Lib/fontTools/misc/plistlib/__init__.py

```diff
@@ -521,13 +571,13 @@ def loads(value, use_builtin_types=None, dict_type=dict):
 
 
 def dump(
-    value,
-    fp,
-    sort_keys=True,
-    skipkeys=False,
-    use_builtin_types=None,
-    pretty_print=True,
-):
+    value: PlistEncodable,
+    fp: IO[Any],
```

Comment:
why `IO[Any]`? The load method takes bytes. Does this work with str too? If so, why can't you use a `IO[Union[str, bytes]]`?

Reply:
Hm, dunno, but all tests use BytesIO so I'll change it to bytes.